src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveAttributeCompletionItemProvider.cs

```diff
@@ -141,6 +141,9 @@ public override IReadOnlyList<RazorCompletionItem> GetCompletionItems(RazorSynta
                     insertText = insertText.Substring(0, insertText.Length - 3);
                 }
 
+                // Strip off the @ from the insertion text.
+                insertText = insertText.Substring(1);
```

Comment:
Should we sanity check that we're trimming the right thing? I feel like this also deserve a comment explaining why specifically we need to do this.

Reply:
Sure can add both